packages/cli/src/executions/parse-range-query.middleware.ts

```diff
@@ -44,7 +44,7 @@ export const parseRangeQuery = (
 
 			if (jsonFilter.waitTill) jsonFilter.waitTill = Boolean(jsonFilter.waitTill);
 
-			if (!isValid(jsonFilter)) throw new ApplicationError('Query does not match schema');
+			if (!isValid(jsonFilter)) throw new UnexpectedError('Query does not match schema');
```

Comment:
This input is coming from users --> `UserError`

Reply:
This is coming from our own frontend. Cases like I'd think this is unexpected and we'd want to report it?